Assets/Scripts/FieldElementsDisabler.cs

```diff
@@ -0,0 +1,21 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+public class FieldElementsDisabler : MonoBehaviour
+{
+    [SerializeField] private List<GameObject> _circles;
+
```

Comment:
you dont need to have separate lists for crosses and circles, just store them in onel list.